### PR TITLE
Move add labels button out of the info text

### PR DESCRIPTION
**Pull request by ubergesundheit**
Towards https://github.com/giantswarm/giantswarm/issues/11424

Here is a screenshot of how it'll look after merging this PR:

image

### CODE REVIEW

File: src/components/Cluster/ClusterDetail/ClusterLabels/ClusterLabels.tsx [hunk 1]
```diff
@@ -80,15 +84,15 @@ const ClusterLabels: FC<IClusterLabelsProps> = ({
       <LabelsTitle>Labels:</LabelsTitle>
       {noLabels ? (
         <NoLabels>
-          This cluster has no labels. You can add a label by clicking the{' '}
-          <EditLabelTooltip
+          This cluster has no labels. You can add a label by clicking the Add
+          button.
```
Comment (marians) [inline src/components/Cluster/ClusterDetail/ClusterLabels/ClusterLabels.tsx:88]:
```suggestion
          This cluster has no labels.
```

Comment (marians) [reply src/components/Cluster/ClusterDetail/ClusterLabels/ClusterLabels.tsx:88]: Please also change the button label from `Add` to `Add label`

**Comment by ubergesundheit** [comment]
image

image

File: src/components/Cluster/ClusterDetail/ClusterLabels/EditLabelTooltip.tsx [hunk 2]
```diff
@@ -138,14 +140,14 @@ const EditLabelTooltip: FC<IEditLabelTooltip> = ({
...
-    <EditLabelTooltipWrapper ref={divElement}>
+    <EditLabelTooltipWrapper ref={divElement} className={className}>
       {label === '' ? (
         <AddLabelButton
           disabled={!allowInteraction || currentlyEditing}
           onClick={open}
           data-testid='add-label-button'
         >
-          Add
+          <i className='fa fa-add-circle' /> Add
```
Comment (marians) [inline src/components/Cluster/ClusterDetail/ClusterLabels/EditLabelTooltip.tsx:150]:
```suggestion
          <i className='fa fa-add-circle' /> Add label
```

**Comment by ubergesundheit** [comment]
image

image

**Pull request merged**